.github/workflows/windows.yml

```diff
@@ -60,3 +60,54 @@ jobs:
             set PATH=%PROJ_DIR%\bin;%PATH%
             call %GITHUB_WORKSPACE%\test\postinstall\test_cmake.bat %PROJ_DIR%
             proj
+
+  MSYS2:
+    runs-on: windows-latest
+    if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]')"
+
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    env:
+        CMAKE_GENERATOR: "MSYS Makefiles"
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install:  >-
+            git
+            make
+            mingw-w64-x86_64-cmake
+            mingw-w64-x86_64-curl
+            mingw-w64-x86_64-gcc
+            mingw-w64-x86_64-libtiff
+            mingw-w64-x86_64-sqlite3
+
+      - name: Build
+        run: |
+            PROJ_BUILD=${GITHUB_WORKSPACE}/build
+            PROJ_DIR=${GITHUB_WORKSPACE}/proj_dir
+            mkdir ${PROJ_BUILD}
+            cd ${PROJ_BUILD}
+            cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX="${PROJ_DIR}" ..
+            make -j 2
+            make install
+            ls ${PROJ_DIR}/bin
+
+      - name: Run tests
+        run: |
+            PROJ_BUILD=${GITHUB_WORKSPACE}/build
+            PROJ_DIR=${GITHUB_WORKSPACE}/proj_dir
+            export PROJ_LIB=${PROJ_DIR}/share/proj
```

Comment:
do we actually need this ? This should be hard coded in the binary . The same also applies for the MSVC build actually

Reply:
I had assumed there was a reason you did this for the MSVC build above, but let's drop these and see how it goes...